app/src/main/java/com/alphawallet/app/repository/TransactionRepository.java

```diff
@@ -15,6 +16,11 @@
 
 import org.web3j.protocol.Web3j;
 import org.web3j.protocol.core.methods.response.EthSendTransaction;
+import org.web3j.protocol.core.methods.response.EthTransaction;
+import org.web3j.protocol.core.methods.response.TransactionReceipt;
+import org.web3j.rlp.RlpEncoder;
+import org.web3j.rlp.RlpList;
+import org.web3j.rlp.RlpType;
 import org.web3j.utils.Numeric;
 
 import java.math.BigInteger;
```

Comment:
Could you do a 'compare with origin/master' and sync to that?

Reply:
Sorry I missed -es.   
Compared and synced TransactionRepository with master